### DISCUSSION TITLE
Issue with Designer since RC1

### DISCUSSION THREAD

**Discussion by dbriard** [AvaloniaUI/Avalonia#11671] [General]
Hi, 

Before RC1, I was able to open a xaml in VS2022 with the Designer working, and to change the Xaml to update the designer. Something I had to make two consecutives changes in Xaml, the first one make the designer to display nothing, and the second worked fine.

But now since RC1, the designer display the view only once... an when I change something in Xaml, the designer no not update anymore and display a blank view (shadows with transparent content).

Is it just me, or may be there is an update of the designer to install?

**Comment by MikeCodesDotNET** [comment]
Is it a new project? 

I think it's defaults to the browser, and it won't work (known issue). 
<img width="297" alt="image" src="https://github.com/AvaloniaUI/Avalonia/assets/552074/90c593eb-d4a2-48e9-aa58-3fcc7d4867d3">

**Reply by dbriard** [reply]
It is not a new project, it is a project I am working on since preview 4 and I use the Designer with Desktop backend.

**I just try to create a new project with RC1 using "dotnet new avalonia.xplat"**, set Desktop app as startup project, run it, close it, then open MainView... the Designer display the view, but then if I change something in xaml such as adding two whitespaces, the Designer display empty view and never refresh again...

Tested on two different computers. No issue from preview 4 to preview 8. Only RC1.

I have an old project with preview 7, and Designer work fine with it.

I tried to reinstall the VS extensions and that do not solve the issue.

There is no error or warning in the Avalonia Diagnostic window.
